change/@office-iss-react-native-win32-87a19e92-c01c-4f88-b7ad-a1ab461fb380.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The change files seems to be wrong: they are missing a change for the RNW, while have files where we had no changes. This is due to the beachball changes. To fix it please do the following: remove these three change files and run the yarn change with the explicit upstream branch. E.g. `yarn change --branch upstream/master`